Chargebee/Classes/ReceiptValidation/CBReceiptValidationManager.swift

```diff
@@ -37,8 +37,13 @@ struct CBValidateReceiptWrapper: Decodable {
         case inAppSubscription = "in_app_subscription"
     }
 }
+// MARK: - Subscription
+public struct PurchaseSubscription: Decodable {
+    public let status: Bool
+    public let subscriptionDetails: CBValidateReceipt
```

Comment:
Any reason that we want to have `subscriptionDetails` here? IMO this struct is related to subscription and any details can be here itself. Check with `NonSubscription`